### PR TITLE
Various updates to follow up recent PRs

### PR DESCRIPTION
**Pull request by razor-x**
- refactor: Remove newline
- feat: Update seamapi
- refactor: Update UPSTREAM comments
- fix: Use AccessCodesListRequest type from seamapi
- refactor: Re-enable strict-boolean-expressions
- refactor: Remove commented out code
- refactor: Return null directly
- refactor: Always return conditional components in block
- refactor: Remove TODO
- test: Remove isomorphic-fetch
- refactor: Add VisualStudioCode section to gitignore
- fix: Update useAccessCode hook to match standard API

### CODE REVIEW

**Review by mikewuu — APPROVED** [review]

**Comment by razor-x** [comment]
@mikewuu Pushed a few more minor things after your review, will merge to but feel free to comment if you have any feedbac:

- [`2134d91` (#145)](https://github.com/seamapi/react/pull/145/commits/2134d91b8a3261f733aa6518c8d0dca210e2c25f)
- [`7f69b3f` (#145)](https://github.com/seamapi/react/pull/145/commits/7f69b3f1cc193aea4d5ba7294d4b966bf4e8f0b5)
- [`b977089` (#145)](https://github.com/seamapi/react/pull/145/commits/b9770894a31d8744548b76ddb109cb6776f80dd2)
- [`503127a` (#145)](https://github.com/seamapi/react/pull/145/commits/503127a540704acf1a7e4adfc1ba9c8fdc0cd9c2)
-

**Pull request merged**
